### DISCUSSION TITLE
Translation i18n language support

### DISCUSSION THREAD

**Discussion by syntrydy** [josdejong/svelte-jsoneditor#336] [Ideas]
I want to customize the tooltip text that show up when the cursor over a specific element from the toolbar?

![image](https://github.com/josdejong/svelte-jsoneditor/assets/7513418/1cfffd9a-14f2-447f-b22f-4c0b733961e2)

**Comment by josdejong** [comment]
There is no option for that right now. I think the right approach would be to introduce language support (i18n) so you can customize all texts in the editor. See also #292.

**Reply by josdejong** [reply]
Anyone interested in helping to implement language support? Help would be welcome.
